src/Style/web/BaseStyle.js

```diff
@@ -0,0 +1,25 @@
+import { createGlobalStyle } from 'styled-components';
```

Comment:
I really don't know which folder this should go since `injectGlobal` is not available anymore and this returns a component so it can't go in `theme`. Or else we figure out a way to restructure `theme` itself. @lakshyaranganath suggestions here 🙈

Reply:
I would still keep it in 
`theme/web/BaseStyles`